### DISCUSSION TITLE
Postgres HA

### DISCUSSION THREAD

**Discussion by shubhra27** [artifacthub/hub#1856] [Q&A] [answered]
Hi,

I am trying to install bitnami's postgres HA helm chart to my kubernetes environment.
The database version that is available as default is 11.5. I need to use 12.x + version. When I am trying to increment the database's docker image's version, the containers are crashing. I am getting below error:

ERROR:  failed to authenticate with backend using SCRAM
DETAIL:  valid password not found

Note: I haven't changed any default password values.

Any help/suggestion most welcome.

**Comment by tegioz** [comment, marked as answer]
Hi @shubhra27 👋 

For issues with the content listed on `artifacthub.io` it's best to contact the particular project directly. 

This link may help for issues related to Bitnami charts: https://github.com/bitnami/charts/issues

**Reply by shubhra27** [reply]
Thanks for redirecting
